corehq/apps/es/management/commands/make_elastic_migration.py

```diff
@@ -213,11 +240,18 @@ def adapter_and_properties_type(self, value):
         properties = all_properties = adapter.mapping["properties"]
         if delim:
             properties = {}
+            nested_delim = '.'
             for name in property_names.split(","):
                 if not name:
                     continue
+
```

Comment:
Can we add some tests to verify the updated functionality? 
There already are some tests for this, so they should be pretty straightforward to add.
https://github.com/dimagi/commcare-hq/blob/f021e81b1aadd4a806c97c38155c0e6b19e4f40d/corehq/apps/es/tests/test_command_make_elastic_migration.py#L192

Reply:
Will do!

Reply:
See [commit](https://github.com/dimagi/commcare-hq/pull/32879/commits/da8d13efba39c3d951a13a74476488d4f51136bc).